src/view/styles/Simulator.css

```diff
@@ -63,12 +64,9 @@
     }
 }
 
-.simulator {
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
-    max-width: 700px;
-    max-height: 700px;
-    margin-left: auto;
-    margin-right: auto;
+.microbit-container {
+    max-width: 350px;
```

Comment:
Why is this not also 100% as cpx?

Reply:
There is already some max sizing in the default cpx styling, but not on micro-bit